action/src/main.ts

```diff
@@ -1,9 +1,18 @@
 import * as core from '@actions/core';
 import { Utils } from './utils';
 
+
 async function main() {
     try {
         core.startGroup('Frogbot');
+        //validate JF_URL
+        let jfrogUrl: string = process.env.JF_URL?? '';
+        if (!jfrogUrl) {
```

Comment:
obviously can be in both - but if the following function is validatePlatformUrl - shouldn't we validate it there?